docs/self-managed/connectors-deployment/connectors-configuration.md

```diff
@@ -223,75 +223,71 @@ java -cp 'connector-runtime-application-VERSION-with-dependencies.jar:...:my-sec
 
 ## Multi-tenancy
 
-The Connector Runtime supports multiple tenants for inbound and outbound Connectors.
+The Connector Runtime supports multiple tenants for inbound and outbound Connectors. These are configurable in [Identity](/self-managed/identity/user-guide/tenants/managing-tenants.md).
+
 A single Connector Runtime can serve a single tenant or can be configured to serve
-multiple tenants. By default, the runtime uses the `<default>` tenant ID for all
-Zeebe related operations like handling Jobs and publishing Messages.
+multiple tenants. By default, the runtime uses the default tenant ID for all
```

Comment:
```suggestion
multiple tenants. By default, the runtime uses the tenant ID `<default>` for all
```

I think this is one of the references where being explicit about the default being `<default>` is clearer.